CMakeLists.txt

```diff
@@ -1,476 +1,476 @@
-#generate_export_header(${DRV_NAME})
-target_compile_definitions(${DRV_NAME} PRIVATE "DRIVER_BUILD")
-add_dependencies(${DRV_NAME} dsneditor)
-include_directories(${ODBC_INC} ${DRV_SRC_DIR} ${LIBCURL_INC_PATH}
-	${UJSON4C_INC} ${CTIMESTAMP_PATH_SRC} ${TINYCBOR_INC}
-	${DSNEDITOR_INC_PATH})
-target_link_libraries(${DRV_NAME} odbccp32 legacy_stdio_definitions
-	${DSNBND_LIB_BIN_DIR_BASE}-$<CONFIG>/esdsnbnd${BARCH}${CMAKE_IMPORT_LIBRARY_SUFFIX}
-	zlib libcurl ${LIBCURL_WIN_LIBS})
-
-
-#
-# Set up the testing project/target
-#
-enable_testing()
-# ... and testing directory to build
-add_subdirectory(test)
-
-
-#
-# Set up the instalation
-#
-set(INSTALL_DIR
-	${DRIVER_BASE_NAME}-${DRV_VERSION}${VERSION_QUALIFIER}-${PLAT_ARCH}
-	# this must remain a STRING (i.e not PATH), otherwise CPACK won't work
-	CACHE STRING
-	"Directory to install the driver files into")
-
-# this will cause the 'install' to no longer depend on 'all', which avoids
-# building the tests, but this also means that the library must be built
-# before building the 'install' target.
-set(CMAKE_SKIP_INSTALL_ALL_DEPENDENCY TRUE)
-# add instalation project/target
-install(FILES
-	LICENSE.rtf LICENSE.txt ${CMAKE_BINARY_DIR}/NOTICE.txt
-	DESTINATION ${INSTALL_DIR})
-# add libcurl if build dynamically
-# add editor DLLs
-install(FILES
-	${DSNBND_LIB_BIN_DIR_BASE}-$<CONFIG>/esdsnedt${CMAKE_SHARED_LIBRARY_SUFFIX}
-	${DSNBND_LIB_BIN_DIR_BASE}-$<CONFIG>/esdsnbnd${BARCH}${CMAKE_SHARED_LIBRARY_SUFFIX}
-	DESTINATION ${INSTALL_DIR})
-install(TARGETS ${DRV_NAME}
-	DESTINATION ${INSTALL_DIR})
-
-#
-# Set up the packaging
-#
-set(CPACK_GENERATOR "ZIP")
-# don't build ALL (but see CMAKE_SKIP_INSTALL_ALL_DEPENDENCY comment)
-set(CMAKE_SKIP_PACKAGE_ALL_DEPENDENCY TRUE)
-set(CPACK_PACKAGE_FILE_NAME
-	${DRIVER_BASE_NAME}-${DRV_VERSION}${VERSION_QUALIFIER}-${PLAT_ARCH})
-include(CPack)
-
-
-#
-# Generation summary
-#
-message("  ***** <Generation summary>")
-message("Driver source files: ${DRV_SRC}")
-message("Driver compile flags: ${CMAKE_C_FLAGS}")
-message("Driver include paths: ${ODBC_INC} ${DRV_SRC_DIR}"
-	"${LIBCURL_INC_PATH} ${UJSON4C_INC} ${CTIMESTAMP_PATH_SRC}"
-	"${DSNEDITOR_INC_PATH}")
-# there's no var for this
-message("Driver link libraries: odbccp32 legacy_stdio_definitions esdsnbnd"
-	"libcurl ${LIBCURL_WIN_LIBS}")
-message("Driver install target: ${INSTALL_DIR}.")
-message("Driver packaging target: ${CPACK_PACKAGE_FILE_NAME}.")
-message("   ***** </Generation summary>")
-
-# vim: set noet fenc=utf-8 ff=dos sts=0 sw=4 ts=4 tw=78 :
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+# 3.1.0: CMAKE_GENERATOR_PLATFORM
+cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)
+
+set(DRIVER_BASE_NAME elasticodbc CACHE STRING
+	"The base name of the driver")
+
+# driver's version
+set(DRV_VERSION 7.12.2)

```

Comment:
The version change looks good, but it seems file's line endings have been been pushed along?

Reply:
Ah dang, thanks for the heads-up on that. Neovim!